integration/utils/follower.go

```diff
@@ -148,6 +160,10 @@ Loop:
 					}
 				} else {
 					blockUnknownTxs++
+					f.logger.Trace().
+						Uint64("blockHeight", block.Height).
+						Hex("txID", tx.Bytes()).
+						Msg("block unknown TX") // should never happen
```

Comment:
comment is misleading

Reply:
Thanks for the comment! Why is the comment misleading? Are you expecting `block unknown TX` to show up?